share/spice/just_delete_me/content.handlebars

```diff
@@ -0,0 +1,15 @@
+{{#if notes}}
+  <p>
+    {{notes}}
```

Comment:
I was following the convention on the justdeleteme page, which as you say, can be a bit vague at times.